pipeline/scripts/10-collectSamplesFilled.R

```diff
@@ -24,10 +24,10 @@ ppm <- as.numeric(cmd_args[6])
 #z_score <- 0
 
 object.files = list.files(paste(outdir, "9-samplePeaksFilled", sep="/"), full.names=TRUE, pattern=scanmode)
-
 outlist.tot=NULL
 for (i in 1:length(object.files)) {
   load(object.files[i])
+  print(print(object.files[i]))
```

Comment:
it is a remnant of a test. I am in conflict if we should keep it.
This prints all the files from step 9 that are loaded by script 10. if script 10 starts too quickly, not all files are loaded and are directly correlated to the missing m/z values from this bug. 

It just prints the filenames to the .o (output) logging file, which we delete after a month when the run is successful. therefore I think we should keep it for now. When something goes wrong, it is then easier to pinpoint the problem, because it is normally not logged.

I will add this to my azure ticket, of code we need to remove in the next version if all keeps working fine (this also includes the missing m/z file and missing m/z message in the end mail)

Reply:
Thanks for the clarification. This seems fine.